mantidproject/mantid
# Change D33's pixels order

**Description of work.**


This PR changes the underlying ordering of pixels in D33 detectors, so they match the physical reality.
Now, the tubes in the rear, front top and front bottom detector banks are horizontal, those on the left and right banks are vertical.

**To test:**
- Open `ExternalData/Testing/Data/UnitTest/ILL/D33/002294.nxs` and show the instrument view
- If possible, check against the previous version of Mantid that the image shown is the same
- In the `Draw` tab, pick the `Select tube/bank` option and check on each panel that the tubes are correctly oriented 


WIP for #26906 





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
